lib/cmd/fh3/undeploy.js

```diff
@@ -0,0 +1,50 @@
+/* globals i18n */
+module.exports = undeploy;
+undeploy.undeploy = undeploy;
```

Comment:
[Style] Assigning field to function it's not recommended javascript practice. Can we avoid that? Here we assign function to filed of this function. Why we need it?